codespell_lib/data/dictionary.txt

```diff
@@ -40749,6 +40749,8 @@ numberring->numbering
 numberrs->numbers
 numberss->numbers
 numbert->number
+numbes->numbers
```

Comment:
This could be a typo of other things, but `numbers` is more closer. Let me know what to do here.

Reply:
Typically you'd write:
```
numbes->numbers, numbs, numbed,
```

Reply:
Done. PTAL. :)